osbs/build/spec.py

```diff
@@ -147,6 +147,7 @@ class CommonSpec(BuildTypeSpec):
     name = BuildIDParam()
     yum_repourls = BuildParam("yum_repourls")
     use_auth = BuildParam("use_auth", allow_none=True)
+    build_image = BuildParam('build_image')
```

Comment:
It appears to me that the defafault= value is overwritten by the first call to `set_params`, namely to `None` if no value is passed.